helm/charts/hpe-csi-driver/README.md

```diff
@@ -27,6 +27,7 @@ The following table lists the configurable parameters of the chart and their def
 | disable.alletra9000       | Disable HPE Alletra 9000 CSP `Service`.                                | false            |
 | disableNodeConformance    | Disable automatic installation of iSCSI, multipath and NFS packages.   | false            |
 | disableNodeConfiguration  | Disables node conformance and configuration.`*`                        | false            |
+| disableNodePanic          | Disables node panic and allows the reboot of the node                  | false            |
```

Comment:
Change the description to `Enables node kernel panic when processes are hung (waiting for IO) for more than 5 minutes`